config/start_py_test.py

```diff
@@ -0,0 +1,30 @@
+# pylint: skip-file
```

Comment:
I do not really understand this test. It checks for the line that is already in `start.py` file, so this test will pass - why do we need it then?